sqlitecluster/SQLiteNode.cpp

```diff
@@ -1259,11 +1264,31 @@ void SQLiteNode::_onMESSAGE(Peer* peer, const SData& message) {
             SINFO("Got STANDUP_RESPONSE but not STANDINGUP. Probably a late message, ignoring.");
         }
     } else if (SIEquals(message.methodLine, "SYNCHRONIZE")) {
-        // SYNCHRONIZE: Sent by a node in the SEARCHING state to a peer that has new commits. Respond with a
-        // SYNCHRONIZE_RESPONSE containing all COMMITs the requesting peer lacks.
-        SData response("SYNCHRONIZE_RESPONSE");
-        _queueSynchronize(peer, response, false);
-        _sendToPeer(peer, response);
+        // If we're MASTERING or SLAVING, we'll let worker threads handle SYNCHRONIOZATION messages.
+        if (_state == MASTERING || _state == SLAVING) {
```

Comment:
Do you want to do STANDINGDOWN here?

Reply:
Discussed offline, this seems to add little benefit at the risk of subtle bugs.